sdk/python/kfp/compiler/v2_compat.py

```diff
@@ -52,6 +52,12 @@ def update_op(op: dsl.ContainerOp,
   op.add_volume_mount(
       k8s_client.V1VolumeMount(name='kfp-launcher', mount_path='/kfp-launcher'))
 
+  # op.command + op.args will have the following sections:
+  # 1. args passed to kfp-launcher
+  # 2. a separator "--"
+  # 3. parameters in format "key1=value1", "key2=value2", ...
```

Comment:
Maybe worth clarifying what are the examples of these arguments, and what separate them from 1 (args passed to launcher) and 5 (original user program args)?

Reply:
Good point, let me add more docs